### DISCUSSION TITLE
Raditation advancement settings/changes

### DISCUSSION THREAD

**Discussion by Daimonin** [FakeFishGames/Barotrauma#6291] [Ideas and suggestions]
THE PROBLEM:
I like the radiation in the campaign map, but... It kinda negates a bunch of other stuff, like the outposts developing, new settlements, or abandoned outposts getting rebuilt after clear. Because by the time that happens, odds are the outpost is already within the radiation belt, and effectively lost to you. This becomes especially a problem in the later stages of the campaign, when there is very few if any settled locations, but activating sonar beacons/clearing abandoned outposts, and waiting for them to be established is impossible because the radiation is probably going to get there before the colonists do.

Besides that, a more minor issue, it... sorta breaks immersion when you realize the entire civilization is doomed, because every single outpost is engulfed by radiation. Even if outposts have stronger, rad resistant hulls, that still means the people are doomed, because the subs to transport goods and acquire resources don't.

SOLUTION:
I think it would be nice if rather then just radiation on/off when starting a campaign, there was a slider that affects how fast the radiation front moves across the map. This way, the players still get the "being pushed forward" effect, but can customize it to their preferred speed of play, and how much they want to interact with the whole outpost development/settling/clearing system.

ALTERNATIVE SOLUTION:
Radiation progress slows down as you get deeper, so while you get pushed out of the heavily populated biomes fairly quickly, you get more time to actually interact with the outpost development/settling/clearing further down, when the map is more empty.

ALTERNATIVE SOLUTION 2:
Rather then a solid blanket that covers the left side of the map forever, make it pulsing waves of radiation that travel deeper. Getting caught in one sucks, and you generally want to stay ahead of them, but you can stock up on anti-rads and push back through the wave into the clear area behind it, or pay a station docking fees to remain protected in their hull while the wave advances past. Either way, you get a bit of breathing space at a cost, and there's another wave coming not too far away so better get pushing deeper again, or start saving up money/anti-rads for when it hits.

**Comment by James-E-A** [comment]
another solution: (some) people won't just sit around to get microwaved. what if every City that abandons due to radiation will select another random node within 7 hops of the radiation frontier, and

- if it's uninhabited but hospitable (no adjacent abyss monster, not currently an Abandoned Outpost that hasn't been cleared, not currently irradiated), it instantly becomes a Habitation Outpost
- if it's currently a Habitation Outpost, it has a 50% chance to instantly become a City, otherwise it runs this same algorithm on a random adjacent node after a 1-mission delay
- if it's currently a Research or Military outpost, it has a 25% chance to instantly become a City, and a 50% (independent) chance of running this same algorithm on a random adjacent node after a 1-mission delay
- if it's currently a City, it has a 25% chance to run this algorithm on a random adjacent node after a 1-mission delay, with the caveat that uninhabited nodes will become Research or Military outposts

add to that some reputation inheritance (as the migrants tell their new compatriots how helpful/awful you are), and something like that should be a reasonable draft

**Comment by MaaiG** [comment]
I believe that the effects of radiation should depend on the resulting dose of radiation! So, if you’re at the surface for a long time and you get a high dose of radiation, the radiation forces you to go deep. By staying deep, out of the radiation zone, you will be able to climb back to the surface in sufficient time, as your dose limit has increased again.
Now, radiation is illogical. It’s always a one-way street. All those who come to Europe are suicide? What’s the point of such journeys?
